package.json

```diff
@@ -31,6 +31,7 @@
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "react-helmet": "^5.2.0",
+    "react-helmet-async": "^0.2.0",
```

Comment:
nu folosim v1.0.2?

Reply:
I will test that version and update the pr